src/tools/cephfs/cephfs-shell

```diff
@@ -243,25 +244,58 @@ def copy_to_local(shell, remote_path, local_path):
         fd.close()
 
 
-def dirwalk(dir_name):
+def dirwalk(dir_name, giveDirs = 0):
```

Comment:
could you move this change to a separated commit? also i failed to find the caller site where `giveDirs` is passed in. am i missing something? and we don't use "camelCase" naming convention in python for naming variables or parameters.